examples/guides/mp-restful-webservice/src/main/docs/index.adoc

```diff
@@ -0,0 +1,487 @@
+
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+:adoc-dir: {guides-dir}/mp-restful-webservice/src/main/docs
+:java-base: {adoc-dir}/../java/io/helidon/guides/mp/restfulwebservice
+:greet-app: {java-base}/GreetApplication.java
+:main-class: {java-base}/Main.java
+:greet-resource-class: {java-base}/GreetResource.java
+:greeting-message-class: {java-base}/GreetingMessage.java
+:greet-application-class: {java-base}/GreetApplication.java
+:health-resource-class: {java-base}/HealthResource.java
+:pom: {adoc-dir}/../../../pom.xml
+:meta-inf-dir: {adoc-dir}/../resources/META-INF
+:beans-xml: {meta-inf-dir}/beans.xml
+:config-file: {meta-inf-dir}/microprofile-config.properties
+:logging-properties-file: {adoc-dir}/../resources/logging.properties
+:se-guide-adoc: {guides-dir}/se-restful-webservice/src/main/docs/index.adoc
+
+
```

Comment:
I would like to read through & review this content, but it'll take me some time. By when do you need this review to be completed?